docs/how-to/create-new-event.rst

```diff
@@ -0,0 +1,60 @@
+How to create a new Open edX Event
+==================================
+
+The mechanisms implemented by the Open edX Events library are supported and maintained by the Open edX community.
+Therefore, we've put together a guide on how to add a new event to have an effective contribution process.
+
+
+1. Propose the new event to the community
+-----------------------------------------
+
+When creating a new event, you must justify its implementation. For example, you could create a post in Discuss,
+send a message through slack or open a new issue in the library repository listing your use cases for it. Or even,
+if you have time, you could accompany your proposal with the implementation of the event to illustrate its behavior.
+
+
+2. Create the data attributes for the event (OEP-49)
+----------------------------------------------------
+
+Events send data attribute instances when triggered. Therefore, when designing your new event definition you must
+decided if an existent data attribute class works for your use case or you must create a new one. If the answer is
+the latter, then try to answer:
+
+- Which attributes of the object are the most relevant?
+- Which type are they?
+- Is any of them optional/required?
+
+And with that information, create the new class justifying each decision. The class created in this step must comply
+with:
+
+- It should be created in the `data.py` file in the corresponding subdomain. Refer to Naming Conventions ADR for more
+  on events subdomains.
+- It should follow the naming conventions specified in...
+
+3. Create the event definition
+------------------------------
+
+Open edX Events are instances of the class OpenEdxPublicSignal, this instance represents the event definition that
+specifies:
+
+- The event type which should follow the conventions in the Naming Conventions ADR.
+- The events' payload, here you must use the class you decided on before.
+
+The definition created in this step must comply with:
+
+- It should be created in the `signals.py` file in the corresponding subdomain. Refer to Naming Conventions ADR for more
```

Comment:
One proposal would be to add even more info about subdomains here. It could include a list of the in-use agreed upon subdomains. It might provide examples of possible other subdomains, but which would need agreement upon first use. Here is [an event design github issue](https://github.com/openedx/openedx-events/issues/72) where Dave is discussing subdomains. I'm hoping we can capture/summarize/consolidate/reduce the amount of discussion around subdomains as we move forward.

Reply:
Sorry for the delay. I'm not sure. I guess I'd start with something small (a couple of sentences?) and only break it out if that seems necessary.

Reply:
I'm going to mark this resolved. I like what you did, and have new comments on the current iteration.